// This is an example of what happens when you panic.
// In this case the code returned to the OS/parent is 101

fn main() {
    panic!("crash and burn");
}
